### ISSUE TITLE
<node_internals> in skipFiles not working in latest node version

### ISSUE THREAD

**Issue by roblourens** [Microsoft/vscode-node-debug2#212]
Due to node using file URIs instead of paths now

**Comment by roblourens** [comment]
node internal scripts: `file:///path.js`, `file:///internal/inspector_async_hook.js`

My file on disk: `file:///Users/roblou/code/vscode-chrome-debug-core/gulpfile.js`

Previously we defined any script with a non-absolute path as a node internal script. I have no idea how we can tell the difference now because node's `path.js` will look the same as a file at `/path.js`.

Maybe we just need to keep a list? Or we can assume any `file:///xyz.js` or `file:///internal/xyz.js` path is a node internal but that doesn't sound safe.

**Comment by weinand** [comment]
@roblourens yes, we have bugs in that area and I'm currently looking into this.
What is your exact use case?

**Comment by roblourens** [comment]
I'm not sure what you mean, what are you looking into? I'm talking about the paths coming from node, that we send a "blackbox scripts" regex against, this is just a node-debug2 feature.

**Comment by weinand** [comment]
Oh, I misunderstood... 

Here what I thought this bug is about:

We should not try to convert a pseudo-relative path like `<node-internals>/foo.js` into a file uri because they are not able to preserve relative paths. Currently this happens when running DA in the EH. My fixes for #61129 and #60667 tried to address this.

After re-reading the issues I see that node.js has changed in latest version...

I'll have to think about this...

**Comment by roblourens** [comment]
I think I should file an issue on node because I think that `file:///path.js` is just not right (that doesn't help your side of the issue)

**Comment by weinand** [comment]
You could suggest to use a `file:///<node-internals>/path.js` syntax... ;-)

**Comment by weinand** [comment]
Just be aware that in file URIs some characters are '%'-encoded which they are not in the blackbox regexes.
